pf9/cluster/commands.py

```diff
@@ -172,7 +182,30 @@ def create(ctx, **kwargs):
 
     # Load active config
     Get(ctx).active_config()
-    Get(ctx).get_token_project()
+    Get(ctx).get_token_project_user_id()
+
+    segment_session = SegmentSession()
+    segment_event_properties = dict(arg_cluster_name=ctx.params['cluster_name'],
+                                    arg_master_ips=ctx.params['master_ip'],
```

Comment:
Can all these args be built in a generic way? It will have to change each time with a CLI change.

Reply:
It can be built, but the problem is that there are so many unwanted things in ctx.params including du_username and passwords that I dont think we should send it outside, then I would have to process based on an ignore list.

Reply:
As per the discussion with you, I will be keeping the code as is to build the args, as it will be difficult to ignore unnecessary things